frontend/src/App.tsx

```diff
@@ -334,42 +367,67 @@ const Form = (props: { model: Model, dispatch: React.Dispatch<Msg> }) => {
 export const EntrypointForms = (props: { model: Model, dispatch: React.Dispatch<Msg> }) => {
 
   const isDisabled = (): string => {
-    if (props.model.contract._tag === "RemoteSuccess") return 'opacity-100 pointer-events-auto'
-    else return 'opacity-50 pointer-events-none'
+    if (props.model.contract._tag === "RemoteSuccess") return ' opacity-100 pointer-events-auto'
+    else return ' opacity-50 pointer-events-none'
   }
 
```

Comment:
I'm not sure why, but now on any successful call that I execute I get a blank page, have you notice this?